### PR TITLE
Update how composite run manifests are built, and some small test work

### PR DESCRIPTION
**Pull request by kedhammar**

### CODE REVIEW

File: taca/element/Element_Runs.py [hunk 1]
```diff
@@ -18,6 +18,84 @@
...
+def get_mask(
+    seq: str,
+    mask_type: str,
+    prefix: str,
+    cycles_used: int | None = None,
+) -> str:
+    """Example usage:
+
+    get_mask("ACGTNNN", "umi", "I1:", None) -> 'I1:N4Y3'
+    get_mask("ACGTNNN", "index", "I2:", 10) -> 'I2:Y4N3N3'
```
Comment (ssjunnebo) [inline taca/element/Element_Runs.py:30]: Should `'I2:Y4N3N3'` be `'I2:Y4N6'`?

**Review by ssjunnebo — APPROVED** [review]
Niice! 👍

File: taca/element/Element_Runs.py (hunk 1 above)
Comment (kedhammar) [reply taca/element/Element_Runs.py:30]: Currently, the function that generates the mask checks if the de-facto number of cycles used is higher than the length of the mask, and if so adds this padding right at the end. I didn't bother merging the groups as you suggest above, but I can if it's preferred or needed :)

Comment (ssjunnebo) [reply taca/element/Element_Runs.py:30]: Ah, I see! If that works it's fine I think 👍

**Pull request merged**